cmd/homi/setup/cmd.go

```diff
@@ -886,14 +890,21 @@ func writeValidatorsAndNodesToFile(validators []*ValidatorInfo, parentDir string
 	}
 }
 
-func writeTestKeys(parentDir string, keys []string) {
+func writeTestKeys(parentDir string, privKeys []*ecdsa.PrivateKey, keys []string) {
 	parentPath := path.Join(outputPath, parentDir)
 	os.MkdirAll(parentPath, os.ModePerm)
 
 	for i, key := range keys {
 		testKeyFilePath := path.Join(parentPath, "testkey"+strconv.Itoa(i+1))
 		ioutil.WriteFile(testKeyFilePath, []byte(key), os.ModePerm)
 		fmt.Println("Created : ", testKeyFilePath)
+
+		pk := privKeys[i]
+		ksPath := path.Join(parentPath, "keystore"+strconv.Itoa(i+1))
+		ks := keystore.NewKeyStore(ksPath, keystore.StandardScryptN, keystore.StandardScryptP)
+		pwdStr := RandStringRunes(10)
```

Comment:
What is the reason for choosing 10 here?

Reply:
Sorry. There is no reason. How about 32? :)
32 is already used in bridge account keystore. It is enough complexity for brute-force attack.

Reply:
http://toplinestrategies.com/blogs/net/how-much-time-needed-crack-password-brute-force

I think 10 is enough. :) I will reflect all RandStringRunes.

Reply:
@aidan-kwon asked me to change a larger one. :)

Reply:
As we talked, I added paramamter for password length. :) Thanks.